cmd/kubeadm/app/cmd/init.go

```diff
@@ -340,43 +365,6 @@ func (d initData) IgnorePreflightErrors() sets.String {
 	return d.ignorePreflightErrors
 }
 
-// CertificateWriteDir returns the path to the certificate folder or the temporary folder path in case of DryRun.
-func (d initData) CertificateWriteDir() string {
```

Comment:
if these getters are removed, how would other packages access the private struct members?